modules/installation/pages/2-pre-installation/1-usb-creation.adoc

```diff
@@ -117,11 +118,16 @@ All the images should return with `OK`. If any fail, please stop and open a bug
 +
 NOTE: `crucible` is undergoing source code changes and will have packages for `yum` and `debian` package managers in the near future.
 +
-* SUSE
+* YUM RPM (SUSE/RedHat)
```

Comment:
This section feels weird. Did you mean to have **YUM RPM (SUSE/RedHat)**

Reply:
I meant just any RPM, so I could ditch the "YUM" bit.